src/main.rs

```diff
@@ -108,8 +111,10 @@ struct Opt {
     work_dir: Option<String>,
 
     // `cargo build` arguments
-    #[structopt(name = "binary", long = "bin")]
+    #[structopt(name = "bin", long = "bin")]
```

Comment:
Could you provide a help text for those too? :)

Also, `--bin` is already taken because it is an argument to `cargo build` which we forward :/

Reply:
haha duh. I finally realized what bin is used for.  I was like why is this not hooked up to anything :) Late night commits...

Im going to pull binary out. Besides the complications for handling suboptions, the name conflict is for the next guy to solve.